Editor/Core/Utilities/Extensions.cs

```diff
@@ -30,5 +30,17 @@ public static bool HasFlag(this ExportPackageOptions input, ExportPackageOptions
         {
             return (input & flag) == flag;
         }
+
+        public static bool TryGetValue<TValue>(this IDictionary<string, object> dict, string key, out TValue value)
```

Comment:
This has the side-effect of boxing structs. Maybe a where `TValue:class` constraint should be used.